src/NHibernate.Test/NHSpecificTest/NH3386/Fixture.cs

```diff
@@ -60,7 +60,7 @@ public void ShouldSupportNonRuntimeExtensionWithoutEntityReference()
 
 	public static class SqlServerFunction
 	{
-		[LinqExtensionMethod]
+		[LinqExtensionMethod(DBOnly = true)]
```

Comment:
It was not working with NH4. Third time I answer that, to the third time you do this objection. This code had start working with a v5 commit having accidentally sort of fixed it. With drawbacks.